### ISSUE TITLE
Change the collation for proposal title and protein name to utf8mb4_unicode_ci

### ISSUE THREAD

**Issue by delageniere** [ispyb/ispyb-database-documentation#23]
Recently we had several troubles while ingesting data from User Portal due to special characters.
This would be fixed if we change the collations for these columns: 
Proposal.title
Protein.name

**Comment by KarlLevik** [comment]
I agree, this makes sense. We actually already have the `utf8mb4_unicode_ci ollation` on our `Proposal.title` at Diamond. But not on `Protein.name`. 

I think these SQL statements will do the job:

`ALTER TABLE Proposal MODIFY title varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL;`  
`ALTER TABLE Protein MODIFY `name` varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL;`

**Comment by delageniere** [comment]
Done with script : 2018_03_13_updateCollations.sql

**Comment by delageniere** [comment]
We face the same type of problems with Macromolecule.name .acronym, .comments
So we will change the collation accordingly.

**Comment by stufisher** [comment]
I cannot agree with acronym. This field is piped to the filesystem where we really do not want special characters.

**Comment by stufisher** [comment]
Sorry to clarify, macromolecule.acronym or protein.acronym? (why are these different tables? separate discussion). If macromolecule i assume this a SAXS issue?

**Comment by delageniere** [comment]
Yes, it is a SAXS issue, and in fact not necessary to change Macromolecule.acronym as it is filtered beforehand.
